src/api/finnKandidatApi.ts

```diff
@@ -1,7 +1,7 @@
 import { ArbeidstidBehov } from '../types/Behov';
 import { Omit } from 'react-router';
 import api from './initialize';
-import Kandidat from '../types/Kandidat';
+import { Kandidat } from '../types/Kandidat';
```

Comment:
På et eller annet tidspunkt klikka default-exporten for `Kandidat` 🙃Så da fjernet jeg defaulten og eksporterte på vanlig vis.